src/util/doc-types.ts

```diff
@@ -26,15 +33,20 @@ export type ParsedAddress = {
 //================================================================================
 // DOCUMENTS
 
+/** Contains data written and signed by an identity. */
 export interface Doc {
+    /** Which document format the doc adheres to, e.g. `es.4`. */
     format: string;
     author: AuthorAddress;
     content: string; // TODO: | null, when we have sparse mode
     contentHash: string;
     //contentLength: number,  // TODO: add for sparse mode, and enforce in the format validator
+    /** When the document should be deleted, as a UNIX timestamp in microseconds. */
     deleteAfter: number | null;
     path: Path;
+    /** Used to verify the authorship of the document. */
     signature: Signature;
+    /** When the document was written, as a UNIX timestamp in microseconds. */
```

Comment:
I would add a note here about microseconds vs milliseconds since it's such a common confusion.

"as a UNIX timestamp in microseconds (millionths of a second, e.g. Date.now() * 1000)"

Reply:
Added.